src/librustdoc/html/static/js/main.js

```diff
@@ -295,30 +292,6 @@ function hideThemeButtonState() {
                 loadSearch();
             }
         },
-        addCrateDropdown: function(crates) {
```

Comment:
Interesting! I was sure it was already moved into `search.js`. Oh well. Now it's done. :)